main/streams/plain_wrapper.c

```diff
@@ -903,6 +903,7 @@ static int php_stdiop_set_option(php_stream *stream, int option, int value, void
 #endif
 				}
 			}
+            return PHP_STREAM_OPTION_RETURN_NOTIMPL;
```

Comment:
Space indent snuck in here.

Reply:
> Space indent snuck in here.

sorry, I will pay attention to the indentation next time